test/integration-test.sh

```diff
@@ -429,16 +472,8 @@ log "hopr addr5: ${addr5} ${native_addr5} ${hopr_addr5}"
 log "hopr addr7: ${addr7} ${native_addr7} ${hopr_addr7}"
 log "hopr addr8: ${addr8} ${native_addr8} ${hopr_addr8}"
 
-# enable register
```

Comment:
Merging this PR with E2E test changes will cause conflicts in other PRs - should be separated so we can reduce conflicts

Reply:
This is expected and not an issue

Reply:
see below. Functionality became a fucntion